fission/function.go

```diff
@@ -41,6 +43,55 @@ func fileSize(filePath string) int64 {
 	return info.Size()
 }
 
+func getFnPkgRef(client *client.Client, fnName string, fileName string) fission.FunctionPackageRef {
+	// TODO fallback to uploading + setting a Package URL
+	checkFileSize(fileName)
+	pkgContents := getPackageContents(fileName)
+	pkgName := fmt.Sprintf("%v-%v", fnName, strings.ToLower(uniuri.NewLen(6)))
+	pkg := &tpr.Package{
+		Metadata: api.ObjectMeta{
+			Name:      pkgName,
+			Namespace: api.NamespaceDefault,
+		},
+		Spec: fission.PackageSpec{
+			Type:    fission.PackageTypeLiteral,
+			Literal: pkgContents,
+		},
+	}
+	_, err := client.PackageCreate(pkg)
+	checkErr(err, "upload package")
+
+	return fission.FunctionPackageRef{
+		PackageRef: fission.PackageRef{
+			Name:      pkgName,
+			Namespace: api.NamespaceDefault,
```

Comment:
minor comment, but use `pkg.Metadata.Namespace`, instead of duplicating the api.NamespaceDefault constant